File: 0242-valid-anagram/solution.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        # if(len(s)!=len(t)):
        #     return False
        # set1=set(s)
        # set2=set(t)
        # if(len(set1)!=len(set2)):
        #     return False
        # for i in s:
        #     if i not in t:
        #         return False
        # return True
        s1=[]
        s2=[]
        for i in s:
            s1.append(i)
        for i in t:
            s2.append(i)
        s1.sort()
        s2.sort()
        if(len(s1)!=len(s2)):
            return False
        for i,j in zip(s1,s2):
            if(i!=j):
                return False
        return True

File: 0242-valid-anagram/solution.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length()!=t.length()) return false;
        vector<char> chr1;
        vector<char> chr2;
        for(int i=0;i<s.length();i++){
            chr1.push_back(s[i]);
            chr2.push_back(t[i]);
        }
        sort(chr1.begin(),chr1.end());
        sort(chr2.begin(),chr2.end());
        for(int i=0;i<chr1.size();i++){
            if(chr1[i]!=chr2[i]) return false;
        }
        return true;
    }
};
